delta/plugins/search/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/Search.scala

```diff
@@ -58,7 +60,7 @@ object Search {
       compositeViews
         .list(
           Pagination.OnePage,
-          CompositeViewSearchParams(deprecated = Some(false), filter = v => UIO.pure(v.id == defaultViewId)),
+          CompositeViewSearchParams(deprecated = Some(false), filter = v => IO.pure(v.id == defaultViewId).toUIO),
```

Comment:
Is this a redundant effect transform? Using the cats `(v.id == defaultViewId)`.pure might work through inference and wouldn't demand any change when that module is also migrated